src/Plugin/PhpUnit.php

```diff
@@ -185,10 +185,12 @@ protected function runConfig($directory, $configFile, $logFormat)
         $cmd       = $this->executable . ' %s %s';
         $success   = $this->builder->executeCommand($cmd, $arguments, $directory);
         $output    = $this->builder->getLastOutput();
+        $covHtmlOk = false;
 
-        if ($fileSystem->exists($this->buildLocation) &&
+        if ($fileSystem->exists($this->buildLocation.'.index.html') &&
```

Comment:
I guess it should be `$fileSystem->exists($this->buildLocation . '/index.html')`

Reply:
Verry likely, yes. Strange, why did I not see this? I run my system like this since a longer time...